float32 x        # x-coordinate of the top-left corner or the center (depending on your use case)
float32 y        # y-coordinate of the top-left corner or the center
float32 width    # Width of the rectangle
float32 height   # Height of the rectangle
int32 label           # Labels of the detected object
float32 prob          # Probability (confidence) of the object detection
string class_name          # Class of the detected object